backend/inspirehep/files/api/s3.py

```diff
@@ -18,14 +18,28 @@ def __init__(self, client, resource):
         self.resource = resource
 
     @staticmethod
-    def get_bucket(key):
+    def get_bucket_for_file_key(key):
         """Return the bucket for the given file key.
 
         :param key: the file key
         :return: bucket: The corresponding bucket.
         """
-        bucket = f"{current_app.config.get('S3_BUCKET_PREFIX')}{key[0]}"
-        return bucket
+        return S3.get_prefixed_bucket(key[0])
+
+    @staticmethod
+    def get_prefixed_bucket(bucket_without_prefix):
```

Comment:
```suggestion
    def get_prefixed_bucket_name(bucket_without_prefix):
```

Reply:
well bucket name, called `bucket` in other parts, that's why I kept it like that.